internal/organizer/db/companies_part8.go

```diff
@@ -12,6 +12,9 @@ func companiesPart8() []domain.CompanyProfile {
 			Type:    domain.CompanyTypeOutsource, // system
 			Name:    "Coherent Solutions",
 			Website: "https://www.coherentsolutions.com/",
+			Careers: "https://www.coherentsolutions.com/careers",
+			About:   "https://www.coherentsolutions.com/about-us",
```

Comment:
Where https://readytotouch.com/private/unstable/wip/organizers/data-population-lists/careers-and-about-and-blog ?